lib/drivers/bq27220.c

```diff
@@ -1,29 +1,85 @@
-
 #include "bq27220.h"
 #include "bq27220_reg.h"
 #include "bq27220_data_memory.h"
 
+_Static_assert(sizeof(Bq27220ControlStatus) == 2, "Incorrect Bq27220ControlStatus structure size");
```

Comment:
better move static asserts near the structure definition, so it would report per-compilation-unit issues with packing/alignment